shell/app/electron_main_delegate.cc

```diff
@@ -440,6 +437,12 @@ base::StringPiece ElectronMainDelegate::GetBrowserV8SnapshotFilename() {
   return ContentMainDelegate::GetBrowserV8SnapshotFilename();
 }
 
+content::ContentClient* ElectronMainDelegate::CreateContentClient() {
+  content_client_ = std::make_unique<ElectronContentClient>();
+  SetContentClient(content_client_.get());
```

Comment:
`SetContentClient` is not needed, it is executed by the caller of this method now https://source.chromium.org/chromium/chromium/src/+/main:content/app/content_main_runner_impl.cc;l=568